edx_exams/apps/api/v1/views.py

```diff
@@ -358,28 +358,25 @@ def get_response(cls, exam, user):
         elif exam.due_date is not None and timezone.now() >= exam.due_date:
             data, grant_access, response_status = {}, True, status.HTTP_200_OK
```

Comment:
todo: I realized it's probably best to remove all the setting of data to {} here and for any cases where access is granted. I can change to not set here and only set below if access is granted.